consensus/service/src/api/peer_api_service.rs

```diff
@@ -252,7 +252,12 @@ impl ConsensusPeerApi for PeerApiService {
                     }
 
                     Err(peer_service_error) => match peer_service_error {
-                        PeerServiceError::Enclave(err) => Err(rpc_enclave_err(err, logger)),
+                        PeerServiceError::Enclave(err) => match err {
+                            Error::Attest(_) => {
+                                Err(rpc_permissions_error("peer_tx_propose", err, logger))
```

Comment:
Note: we may want this to be logged at warning level since its node-to-node attestation error, let's discuss